app/src/LatestResult/index.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable jsx-a11y/media-has-caption */
+import * as React from "react";
+
+// import MoodResult from "../MoodResult";
+import useApi from "../auth/useApi";
+
+import styles from "./styles.module.scss";
+
+const LatestResult = () => {
+  const [latestResult, setLatestResult] = React.useState([]);
+  const [latestMood, setLatestMood] = React.useState([]);
+  const { loading, apiClient } = useApi();
+
+  const loadLatestResult = React.useCallback(
+    async () => setLatestResult(await apiClient.getLatestMood()),
+    [apiClient],
+  );
+
+  React.useEffect(() => {
+    !loading && loadLatestResult();
+  }, [loading, loadLatestResult]);
+
+  if (latestMood == null) {
+    return null;
+  } else {
+    return loading ? null : (
+      <>
+        <h1>Latest Result</h1>
+        <section>
+          <LatestResultist {...{ latestResult }} />
+        </section>
+      </>
+    );
+  }
+};
+
+const LatestResultist = ({ latestResult }) => (
+  <>
+    <div>
+      <ul>
+        <p>You are feeling ...{latestResult.mood}</p>
+        <p>Notes: {latestResult.notes}</p>
+        <p>
```

Comment:
Perhaps this paragraph should be conditional so it is hidden when there is no photo.

Reply:
Good idea, I will also add a conditional like this for my notes section as well!